atlas/data_structure.py

```diff
@@ -330,8 +330,45 @@
             'facility'
         ],
     },
+    'major_unit': {
+        'type': 'string',
+        'allowed': [
+            'arts_sciences',
+            'engineering',
+            'cmci',
+            'undergraduate_education',
+            'continuing_education',
+            'education',
+            'environment_design',
```

Comment:
Just double checking, should this be `environmental_design`?

Reply:
It is `Environment Design` in Jo's docs